### PR TITLE
feat: Port pipeline-health page from old site

### PR DESCRIPTION
**Pull request by alvaromartmart**
🚧 WIP

### CODE REVIEW

File: src/pages/pipeline_health.astro [hunk 1]
```diff
@@ -1,21 +1,296 @@
...
-            width="100%"
-            height="100%"></iframe>
+    <div class="container-fluid main-content w-100 mw-100">
+        <div class="table-responsive">
+            <h2>Pipelines</h2>
+            <table class="table table-hover table-sm">
+                <thead>
+                    <tr>
+                        <th class="small fw-normal text-nowrap">Pipeline Name</th>
+                        <th class="small fw-normal text-nowrap" title="Has at least one release" data-bs-toggle="tooltip" data-bs-placement="top">Released</th>
```
Comment (ewels) [inline src/pages/pipeline_health.astro:18]: Can probably remove all these verbose `data-bs-tooltip` type statements with a small Javascript snippet at the bottom of the file if we want to clean up the HTML a bit.

Not sure if the `small fw-normal text-nowrap` will also work on a parent element?

**Review by ewels — COMMENTED** [review]
Love the way that this is shaping up 👏🏻

File: src/pages/pipeline_health.astro (hunk 1 above)
Comment (alvaromartmart) [reply src/pages/pipeline_health.astro:18]: I haven't payed too much attention to the headers yet, tbh I literally just copy-pasted the elements from https://oldsite.nf-co.re/pipeline_health 😄

**Comment by alvaromartmart** [comment]
@ewels / @mashehu could you `npm run build-pipeline-json` and push the updated `pipelines.json` for me?
Unfortunately I can't as my Github token doesn't have admin privileges over the `nf-core` repos, which is needed to fetch the branch protection rules

**Comment by mashehu** [comment]
hmm, if it was on a branch in this repo, you could invoke the action https://github.com/nf-core/website/actions/workflows/build-json-files-and-md-cache.yml on that branch. but will run it locally for you now.

**Comment by alvaromartmart** [comment]
> hmm, if it was on a branch in this repo, you could invoke the action https://github.com/nf-core/website/actions/workflows/build-json-files-and-md-cache.yml on that branch. but will run it locally for you now.

I didn't even consider creating a branch here, assumed I needed to fork to contribute 😅  but if I'm able to work on a branch directly definitely makes things easier, thanks!

**Comment by alvaromartmart** [comment]
> > hmm, if it was on a branch in this repo, you could invoke the action https://github.com/nf-core/website/actions/workflows/build-json-files-and-md-cache.yml on that branch. but will run it locally for you now.
> 
> I didn't even consider creating a branch here, assumed I needed to fork to contribute 😅 but if I'm able to work on a branch directly definitely makes things easier, thanks!

closing in favor of https://github.com/nf-core/website/pull/2384

File: src/pages/pipeline_health.astro (hunk 1 above)
Comment (ewels) [reply src/pages/pipeline_health.astro:18]: Hah, in that case I literally only have myself to blame for this 😆

**Pull request closed without merging**